scripts/setup_system.sh

```diff
@@ -39,6 +39,19 @@ sudo apt-get -y install \
     gnupg-agent \
     software-properties-common >> /dev/null
 
+# Install Skopeo (ubuntu 18.04+ or Debian 10)
```

Comment:
for simplicity, please assume ubuntu 18. Please add the repo to the sources list before line 25.

Reply:
Okay, so you mean we drop support for Debian? In that case we should adjust the quickstart guide to only mention Ubuntu.

Reply:
we say that we tested on Ubuntu 18, I think that it's enough. We should keep the setup simple.